docs/redirects.csv

```diff
@@ -175,4 +175,5 @@ https://help.expensify.com/articles/new-expensify/expenses/Referral-Program.html
 https://help.expensify.com/articles/new-expensify/workspaces/The-Free-Plan,https://help.expensify.com/new-expensify/hubs/workspaces/
 https://help.expensify.com/new-expensify/hubs/expenses/Connect-a-Bank-Account,https://help.expensify.com/articles/new-expensify/expenses/Connect-a-Business-Bank-Account
 https://help.expensify.com/articles/new-expensify/settings/Security,https://help.expensify.com/articles/new-expensify/settings/Encryption-and-Data-Security
-https://help.expensify.com/articles/expensify-classic/workspaces/reports/Currency,https://help.expensify.com/articles/expensify-classic/workspaces/Currency
\ No newline at end of file
+https://help.expensify.com/articles/expensify-classic/workspaces/reports/Currency,https://help.expensify.com/articles/expensify-classic/workspaces/Currency
```

Comment:
I am confused now actually...apparently the Currency was removed in another PR? Should I delete it from here?

@rushatgabhane @maddylewis

Reply:
@rlinoz currency isn't being modified in this PR